data_summaries/Pure-Dove.json

```diff
@@ -0,0 +1,32 @@
+{
+    "pure_dove": {
+        "Unique Dataset Identifier": "pure_dove",
+        "Dataset Name": "pure_dove",
+        "Paper Title": "",
+        "Dataset URL": "https://huggingface.co/datasets/LDJnr/Pure-Dove",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/LDJnr/Pure-Dove",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "Pure-Dove",
+        "Collection URL": "https://huggingface.co/datasets/LDJnr/Pure-Dove",
+        "Languages": ["English"],
+        "Task Categories": [
+            "Dialog Generation",
+            "Explanation Generation",
+            "Instruction Following",
+            "Instructional Question Answering"
+        ],
+        "Text Sources": ["sharegpt.com", "chat.lmsys.org"],
+        "Model Generated": ["OpenAI GPT-4"],
+        "Format": ["Multi-turn Dialog"],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [""],
```

Comment:
HF description says: "The conversations were sourced from openly datasets such as ShareGPT and ChatBotArena by Lmsys"

Reply:
I see you put those as Text Sources, which is great! Maybe put them here as well since they double as datasets too.